rs/sns/governance/src/lib.rs

```diff
@@ -271,4 +307,36 @@ mod tests {
             }
         }
     }
+
+    #[test]
+    fn test_giant_field_err() {
+        let expected_upper_bound = 12_500;
+
+        let run_test_for_value_of_size = |value_size| {
+            let input_value: String = (0..value_size).map(|_| '🤝').collect();
+            // Sanity check: We construct a string in which each character is encoded as 4 bytes.
+            assert_eq!(input_value.len(), 4 * input_value.chars().count());
+            let observer_err = field_err("foo", input_value.clone(), "bar").unwrap_err();
```

Comment:
```suggestion

            let observer_err = field_err("field_name", input_value.clone(), "ugly").unwrap_err();

```

Reply:
Not sure I see the point to have these dummy values over others?